client/src/actions/tagsSearch.js

```diff
@@ -1,29 +1,29 @@
 import {
```

Comment:
The changes in this file are all renames stemming from making the `TagsSearch` component independent of the `Filters` component.

Reply:
Not really an issue for right now, but this means the `TagsSearch` will not work if there's more than one on the same page (of the same type)

Reply:
Why do you say that?